src/main/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedUberStore.scala

```diff
@@ -214,6 +221,11 @@ class SegmentedUberStore private[segmented] (base: File, eventsPerSegment: Long,
    * Compact readOnlyDirs until they can be compacted no longer.
    */
   private[segmented] def compactAll() {
+    for (toCompact <- segmentedCompactor.findInternalCompactionCandidates(readOnlyDirs)) {
+      val compactedLocation = segmentedCompactor.compactInternally(toCompact)
+      replaceSegment(toCompact, compactedLocation) // mutates readOnlyDirs
+    }
+
```

Comment:
so this will scoop up all the merged segments (but not internally compacted) from the last compaction, right?